### PR TITLE
resize and realign card

### PR DESCRIPTION
**Pull request by potatowagon**

### CODE REVIEW

File: src/main/java/seedu/address/ui/DeadlineTaskListCard.java [hunk 1]
```diff
@@ -33,7 +33,7 @@ public DeadlineTaskListCard(DeadlineTask deadlineTask, int index) {
             indexLabel.setText(IndexPrefix.DEADLINE.getPrefixString() + index + ". ");
             nameLabel.setText(deadlineTask.getName().toString());
             dueLabel.setText(deadlineTask.getDue().toString());
-            finishedLabel.setText(String.valueOf(deadlineTask.isFinished()));
+            //bananaLabel.setText(String.valueOf(deadlineTask.isFinished()));
```
Comment (pyokagan) [inline src/main/java/seedu/address/ui/DeadlineTaskListCard.java]: @potatowagon ????

What's `bananaLabel` and why is it commented out?

**Comment by pyokagan** [comment]
@potatowagon Next time, add a screenshot that shows how the UI looks like after the PR.

For reference:
tasktracker

@potatowagon  No way to shift the "due" dates and priorities to the right like in the mockups?

No changes to the event task card?

**Comment by pyokagan** [comment]
@potatowagon By the way you need to follow project protocol and use the `ReadyForReview` labels. I've been repeating this quite a lot of times already.

**Comment by potatowagon** [comment]
screenshot 52

Task details shifted to the right

screenshot 53
screenshot 54

Task Details always anchored to the right, even with moving panes

**Comment by potatowagon** [comment]
Was thinking since event task pane has lots of verticle space. Also there wont be much space left for event name if start time end time were to be placed on the same line as event name. so didnt do the feature for events

**Review by pyokagan — APPROVED** [review]
OK, Thanks.

Your FXML files has a lot of tabs, but I guess I'll just fix that on my side. :<

**Comment by pyokagan** [comment]
@potatowagon Maybe try doing #63 ?

**Comment by pyokagan** [comment]
Merged as 885f66eef66ca39be9eaa747773c69d031f915d7

**Pull request closed without merging**